### PR TITLE
feat(rbac): update rbac plugin to 1.33.2

### PR DESCRIPTION
**Pull request by christoph-jerolimov**
## Description

update rbac frontend to incl. at least

* https://github.com/backstage/community-plugins/pull/1985
* https://github.com/backstage/community-plugins/pull/2061
* https://github.com/backstage/community-plugins/pull/2067

## Which issue(s) does this PR fix

- Fixes https://issues.redhat.com/browse/RHIDP-4307

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by ciiay — APPROVED** [review]
/lgtm

Works good. Thank you. UI issues will be addressed in this [PR](https://github.com/backstage/community-plugins/pull/2061).

**Comment by ciiay** [comment]
/lgtm

Retested the latest change on OCP v4.15 cluster and it works perfectly. Thank you 👏 

https://github.com/user-attachments/assets/2d6acffc-5c52-4a28-85f8-f059ebe3b43f

**Comment by PatAKnight** [comment]
@christoph-jerolimov I think that the e2e tests are failing due to the mui5 upgrade. I am seeing something similar with the `bulk-import` plugins [update](https://github.com/janus-idp/backstage-showcase/pull/1984). From the looks of it, the txt in the button is no longer wrapped in `<span>` tag anymore and we will need to find a way to update the tests to cover this change. 
After update:
Screenshot from 2024-11-25 12-59-33
Before update:
Screenshot from 2024-11-25 13-01-49

**Comment by christoph-jerolimov** [comment]
@PatAKnight Thanks for that hint!

I updated (see 2nd commit) the UIhelperPO by adding support for MUI v5 buttons.

File: e2e-tests/playwright/support/pageObjects/global-obj.ts [hunk 1]
```diff
@@ -4,7 +4,10 @@ export const waitsObjs = {
 };
 
 export const UIhelperPO = {
-  MuiButtonLabel: 'span[class^="MuiButton-label"]',
+  // MUI v4 buttons are spans with a class starting with MuiButton-label
+  // MUI v5 buttons are HTML buttons with a class containing MuiButton-root
+  MuiButtonLabel:
+    'span[class^="MuiButton-label"],button[class~="MuiButton-root"]',
```
Comment (christoph-jerolimov) [inline e2e-tests/playwright/support/pageObjects/global-obj.ts:10]: fyi: I personally would just use button as selector, but I tried to reduce the risk of regressions with this change.

**Pull request merged**